packages/valory/skills/transaction_settlement_abci/rounds.py

```diff
@@ -243,24 +244,26 @@ def end_block(
         if self.keeper_payload is None:  # pragma: no cover
             return self.synchronized_data, Event.FINALIZATION_FAILED
 
-        verification_status = VerificationStatus(self.keeper_payload["status_value"])
+        verification_status = VerificationStatus(
+            self.keeper_payload.tx_data["status_value"]
```

Comment:
As mentioned above, deliberate breaking change to `keeper_payload` as this is a much more meaningful attribute use.